packages/fx-core/src/component/generator/generator.ts

```diff
@@ -82,7 +84,7 @@ export class Generator {
       destination: destinationPath,
       logProvider: ctx.logProvider,
       zipUrl: sample.link,
```

Comment:
`sample.link ?? sampleConfig.defaultPackageLink`?

Reply:
This logic is in samples.ts. https://github.com/OfficeDev/TeamsFx/blob/7f7c9560f1f5a13bb57eeb7a61feb29da8998d30/packages/fx-core/src/common/samples.ts#L35